conf/app.routes

```diff
@@ -2,7 +2,7 @@
 
 ->          /hmrc-frontend                               hmrcfrontend.Routes
 
-GET         /                                            controllers.IndexController.onPageLoad
```

Comment:
wonder if we should have / redirect to /account?

Reply:
👍